stream-applications-build/pom.xml

```diff
@@ -74,16 +72,16 @@
 
     <dependencyManagement>
         <dependencies>
-            <dependency>
-                <groupId>com.jayway.jsonpath</groupId>
-                <artifactId>json-path</artifactId>
-                <version>${json-path.version}</version>
-            </dependency>
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-compress</artifactId>
                 <version>${commons-compress.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Why has this needed now?

Reply:
There was a compile failure in one module.